src/Onboarding/OnboardingScreen.tsx

```diff
@@ -170,13 +172,15 @@ const skipButtonSpacingTop = Platform.select({
   ios: Spacing.xSmall,
   android: Spacing.xxLarge,
 })
+
 const languageButtonSpacingTop = Platform.select({
   ios: Spacing.small,
   android: Spacing.huge,
 })
-const imageMarginTop = Platform.select({
-  ios: 60,
-  android: 90,
+
+const headerFlex = Platform.select({
+  ios: 0.1,
```

Comment:
A pattern we've been using elsewhere is `Layout.screenHeight * 0.1` or `Layout.oneTenthHeight`. That may not work for this situation, but wanted to make you aware of it just in case.

Reply:
Oh, thanks! Yeah, won't work in this particular case but this is good to know!